pkg/slack/slack.go

```diff
@@ -310,6 +310,18 @@ func BuildJobParams(params string) (map[string]string, error) {
 	}
 	jobParams := make(map[string]string)
 	for _, combinedParam := range splitParams {
+		// Check for DEVSCRIPTS_CONFIG ; if found, we know we're dealing with semicolon separated parameters.
+		if strings.Contains(combinedParam, "DEVSCRIPTS_CONFIG") {
+			// Split by the equal, treat the first part as the key, treat the rest as the value
+			parts := strings.SplitN(combinedParam, "=", 2)
+			if len(parts) == 2 {
+				nParts := strings.Split(parts[1], ";")
+				jobParams[parts[0]] = strings.Join(nParts, "\n")
```

Comment:
This logic seems risky for setting environment variables...
What if someone passes in: `VARIABLE=A=B;C;D=E;F`?